frontend/src/entities/dataset/create/dataset-create-functions.tsx

```diff
@@ -15,14 +15,14 @@
 */
 
 import { IDataset } from '../../../shared/model/dataset.model';
-import { IDatasetFile } from '../../../shared/model/datasetfile.model';
+import { determineScope } from '../dataset.service';
 
-export const createDatasetFromForm = (form: any, files: IDatasetFile[]) => {
+export const createDatasetFromForm = (form: any, projectName: string, username: string): IDataset & Required<Pick<IDataset, 'name' | 'description' | 'type' | 'format' | 'scope'>> => {
     return {
         name: form.name,
         description: form.description,
         type: form.type,
         format: form.format,
-        files: files,
```

Comment:
Why is files removed here?

Reply:
It's no longer part of the dataset object. It doesn't make sense to it exist on this object since it never exists on the object as part of an API call and was clearly appended on as an attempt ease the ergonomics of dealing with files. We don't have a need for that anymore since they're logically separated now. Actually this comment made me realized we're not actually using IDatasetFile anymore and so it can be removed from the project.